### PR TITLE
DOCSP-32483 programmatic tagging script for java sync repo

### PR DESCRIPTION
**Pull request by sarahemlin**
JIRA - https://jira.mongodb.org/browse/DOCSP-32483

Hey @mongoKart - here's the facet.toml file for the java sync driver repo.  Please merge if everything looks good!

Thanks!
Sarah

### CODE REVIEW

File: source/facets.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+[[facets]]
+category = "target_product"
+value = "drivers"
+[[facets]]
+category = "programming_language"
+value = "java"
+[[facets]]
+category = "programming_language"
+value = "java"
```
Comment (mongoKart) [inline source/facets.toml:13]: Do each of these facets (product and language) need to be included twice in this file?

Comment (sarahemlin) [reply source/facets.toml:13]: @mongoKart - good catch!  No, they should not be.  The source file is correct, so I'll need to reach out to Seung & Cassidy.  I think the script is picking up the docs-java-other values, and with the info you shared about that one we might just need to delete the async driver repo from the source file.

Comment (seungpark) [reply source/facets.toml:13]: these doubled values are erroneous! good catch @mongoKart !

https://github.com/10gen/snooty-scripts/pull/8

created an update to match repo path exactly as is in CSV (no partial matching for `docs-java-other` etc)

Comment (sarahemlin) [reply source/facets.toml:13]: Thanks so much, @seungpark!  I'll delete these extra 2 lines so Mike can merge, knowing that the script is now updated so the next time we run it we won't have this problem.

**Comment by sarahemlin** [comment]
@mongoKart - I removed the extra lines so I think we're good to go!

**Review by mongoKart — APPROVED** [review]
lgtm

**Pull request merged**